source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -79,10 +79,13 @@
                             window.location = urlGenerator.action(calificationPeriod, "Evaluations");
                         }
                     }
-                },
-                error: function () {
-                    alert('Fallo interno. Por favor recargue la página.');
                 }
+            })
+            .fail(function () {
+                alert('Fallo interno. Por favor recargue la página.');
```

Comment:
If the session is expired, it is very likely that the server will return a 403 for a POST, and the AJAX promise will be rejected. (At least, jQuery does this.)

It's very likely that it does what you expect, to fail when the return value is not 2xx. (Not sure about 3xx though.)